guides/Getting-Started.md

```diff
@@ -196,8 +199,10 @@ module Test.Main where
 import Prelude
 
 import Euler (answer)
-import Test.Assert (assert)
+import Control.Monad.Eff (Eff)
+import Test.Assert (assert, ASSERT)
 
+main :: forall e. Eff (assert :: ASSERT | e) Unit
```

Comment:
I think this might be a bit confusing as part of a getting started guide. We could instead mention that the compiler will infer the type and we can choose to add it based on the warning. Most editor plugins provide a way to add the inferred type annotation.